src/Fieldset.php

```diff
@@ -203,13 +207,14 @@ public function has(string $elementOrFieldset): bool
      */
     public function get(string $elementOrFieldset): ElementInterface
     {
-        if (! $this->has($elementOrFieldset)) {
+        $element = $this->iterator->get($elementOrFieldset);
```

Comment:
Is this behaviour change safe to perform? 🤔

Reply:
Yes it is:

https://github.com/laminas/laminas-form/blob/b83d29b8fbccb7546c18459745a4875153037973/src/Fieldset.php#L194-L197